test/client.test.js

```diff
@@ -12,7 +12,7 @@ var util = require('util');
 
 var BIND_DN = 'cn=root';
 var BIND_PW = 'secret';
-var SOCKET = '/tmp/.' + uuid();
+var SOCKET = process.platform === 'win32' ? '\\\\.\\pipe\\' + uuid() : '/tmp/.' + uuid();
```

Comment:
```suggestion
var SOCKET = path.join(os.tmpdir(), uuid());
```

Of course, add requires for `path` and `os`.